javaharness/java/arcs/android/BUILD

```diff
@@ -10,22 +10,24 @@ load("@build_bazel_rules_android//android:rules.bzl", "android_library")
 android_library(
     name = "android",
     srcs = glob(["*.java"]),
-    javacopts = ["-Xep:AndroidJdkLibsChecker:OFF"],
-    idl_srcs = glob(["*.aidl"]),
-    idl_import_root = ".",
-    manifest = "AndroidManifest.xml",
     assets = [
         "//shells/pipes-shell:pipes_shell_web",
     ],
-    exports_manifest = 1,
     assets_dir = "pipes_shell_web_dist",
+    exports_manifest = 1,
+    idl_import_root = ".",
+    idl_srcs = glob(["*.aidl"]),
+    javacopts = ["-Xep:AndroidJdkLibsChecker:OFF"],
+    manifest = "AndroidManifest.xml",
+    plugins = [
```

Comment:
Why is this now needed? Didn't it used to get picked up automatically fromthe dagger library?

Reply:
Apparently not. I'm not sure how this used to work, but it was producing build errors now.

Reply:
Does the java library for dagger have an `exported_plugins=` statement? If so, then this `plugins=` shouldn't be needed.

Reply:
Oh, I see. You started using rules_jvm_external, which doesn't automatically setup the dagger plugin system. You should make a dagger library somewhere which imports the core dagger jar, declares a `java_plugin(name = "dagger_plugin", ...)` for the dagger compiler, and have that core dagger library have an `exports_plugins=[":dagger_plugin"]`

Then, you depend on that local definition of dagger, instead of the `@maven//:com_google_dagger_dagger`.